xml/System/Memory`1.xml

```diff
@@ -43,9 +68,17 @@
         <Parameter Name="array" Type="T[]" />
       </Parameters>
       <Docs>
-        <param name="array">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="array">The array from which to create the <see cref="T:System.Memory`1" /> object.</param>
+        <summary>Creates a new <see cref="T:System.Memory`1" /> object over the entirety of a specified array.</summary>
+       <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+If `array` is `null`, this constructor returns a `null` <xref:System.Memory%601> object.
```

Comment:
`Memory<T>` is a struct and can't be null. So stating that it returns a null object could cause confusion. It returns the default struct.